Packs/ExtFilter/pack_metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "ExtFilter",
+    "description": "This transformer enables to write advanced filters.",
```

Comment:
I would suggest expanding a bit more on what kind of advanced filters are available using this script. This is what is shown in the pack description.